src/toil/test/docs/scripts/tutorial_docker.py

```diff
@@ -0,0 +1,14 @@
+from toil.job import Job
```

Comment:
This test is breaking because `dockerCall` isn't declared anywhere.  `dockerCall` has also been updated to `apiDockerCall`.  So I would recommend using: `from toil.lib.docker import apiDockerCall` to use the new version, and changing the args to something like the following:

```
from toil.lib.docker import apiDockerCall
from toil.job import Job
import os

align = Job.wrapJobFn(apiDockerCall,
                      image='quay.io/ucsc_cgl/bwa',
                      working_dir=os.getcwd(),
                      parameters=['index', '/data/reference.fa'])
```